core/src/main/scala/kafka/admin/AclCommand.scala

```diff
@@ -136,7 +200,46 @@ object AclCommand extends Logging {
       )
   }
 
+  private def removeAclWithAdminAPI(opts: AclCommandOptions): Unit = {
+    withAdminClient(opts) { adminClient =>
+      val filterToAcl = getResourceFilterToAcls(opts)
+
+      for ((filter, acls) <- filterToAcl) {
+        if (acls.isEmpty) {
+          if (confirmAction(opts, s"Are you sure you want to delete all ACLs for resource filter `$filter`? (y/n)"))
+            removeAcls(adminClient, acls, filter)
+        } else {
+          if (confirmAction(opts, s"Are you sure you want to remove ACLs: $Newline ${acls.map("\t" + _).mkString(Newline)} $Newline from resource filter `$filter`? (y/n)"))
```

Comment:
I understand that this is proposed in the approved KIP; and sorry for being late with this comment: As far as I can recall in none of the existing command line tools we ask confirmation for a deletion operation (The most extreme one is deleting a topic -- with the default `delete.topic.enable=true` config). If that's actually the case (please remind me if not), is there a particular reason for making an exception here?

Reply:
This is existing functionality.  current AclCommand  asks confirmation for acl deletion.
we need to use "--force" option to disable the confirmation check.

Reply:
Thanks, you are right. This is another example of inconsistency among the command line tools.